src/settings/settings.cpp

```diff
@@ -490,6 +490,10 @@ CombingMode SettingsBaseVirtual::getSettingAsCombingMode(std::string key) const
     {
         return CombingMode::NO_SKIN;
     }
+    if (value == "infill")
```

Comment:
we also need to add an extra option for the "retraction_combing" setting

Reply:
Don't know anything about "retraction_combing" setting.